libraries/ESP8266WiFiMesh/src/TypeConversionFunctions.cpp

```diff
@@ -56,3 +57,89 @@ uint64_t stringToUint64(const String &string, byte base)
   
   return result;
 }
+
+String uint8ArrayToHexString(const uint8_t *uint8Array, uint32_t arrayLength)
+{
+  char hexString[2*arrayLength + 1]; // Each uint8_t will become two characters (00 to FF) and we want a null terminated char array. 
```

Comment:
This could potentially blow up the stack. Instead, consider a local String, reserve() with a precalculated length, and access vie wbuffer() or operator[] or similar.